spec/services/step_nav_publisher_spec.rb

```diff
@@ -29,6 +31,40 @@
         expect(Services.publishing_api).to have_received(:put_content).with(content_id, payload)
       end
     end
+
+    context "marks draft updated" do
+      let(:step_nav) { create(:step_by_step_page_with_steps) }
+
+      it "does not mark submitted for 2i review as draft" do
+        step_nav.update_attributes(:status => "submitted_for_2i", :review_requester_id => review_requester_user.uid, :reviewer_id => reviewer_user.uid)
```

Comment:
Any reason why we're not using `status: "submitted_for_2i"` syntax rather than the rocket?

Reply:
No particular reason.  The usage appears inconsistent throughout the Collections Publisher tests (e.g. `spec/controllers/review_controller_spec.rb` uses rockets for `update_attributes` whereas `spec/factories.rb` uses the colon).

Reply:
Hmm and there's a mixture in this file too. I think generally we prefer the colon syntax unless there's a reason to use the rocket syntax (such as the name of the symbol being incompatible with colon syntax).
I'd be tempted to use the colon syntax here to try to nudge the codebase in the right direction (a cleanup PR later will probably still be needed on the repo as a whole!). It's also a bit less overhead to read.

Reply:
I've updated these lines to use the colon syntax